settings/language-praat.cson

```diff
@@ -0,0 +1,7 @@
+'.source.praat':
+  'editor':
+    'autoIndentOnPaste': true
+    'commentStart': '; '
```

Comment:
According to the Praat documentation:

> Because of its visibility, you are advised to use "#" for comments that structure your script, and ";" perhaps only for "commenting out" a statement, i.e. to temporarily put it before a line that you don't want to execute.